code/game/objects/effects/decals/Cleanable/fuel.dm

```diff
@@ -72,7 +78,7 @@
 
 /obj/effect/decal/cleanable/liquid_fuel/attackby(obj/item/I, mob/user, params)
 	. = ..()
-	if(istype(I, /obj/item/tool/lighter))
+	if(I.damtype == "fire")
```

Comment:
use the damage define BURN

Reply:
didn't i fix this
oh wait that's the other one